src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -738,37 +738,33 @@ private void InitializeWorkers()
                     _hostConfig.LoggerFactory);
             };
 
+            var configFactory = new WorkerConfigFactory(ScriptSettingsManager.Instance.Configuration, _startupLogger);
             var providers = new List<IWorkerProvider>();
             if (!string.IsNullOrEmpty(_language))
             {
-                _startupLogger.LogInformation($"{ScriptConstants.FunctionWorkerRuntimeSettingName} is specified, only {_language} will be enabled");
+                _startupLogger.LogInformation($"{LanguageWorkerConstants.FunctionWorkerRuntimeSettingName} is specified, only {_language} will be enabled");
```

Comment:
nit: can you enclose the params in single quotes?

Reply:
done